src/modules/Builder/pages/BuilderApplet/BuilderApplet.schema.ts

```diff
@@ -227,6 +227,123 @@ export const GyroscopeAndTouchConfigSchema = () => ({
   }),
 });
 
+export const PhrasalTemplateResponseValuePhraseFieldSchema = yup.object({
+  type: yup.mixed().oneOf(['sentence', 'item_response', 'line_break']).required(),
+  text: yup.string().when('type', {
+    is: 'sentence',
+    then(schema) {
+      return (
+        schema.trim()
+          .test("validate there is only one empty sentences", t("fieldRequiredAddToContinue"), function (value, textContext) {
+            const parentPhrasalTemplateResponseValuePhraseSchema = textContext.from?.[1];
+            const fields = parentPhrasalTemplateResponseValuePhraseSchema?.value?.fields || { type: '' };
+
+
+            const isMoreThanOneSentence = fields?.filter(({ type }: { type: string }) => type === 'sentence').length > 1;
```

Comment:
I think a little better to use explicit typing as early as possible, rather than later in the `filter()` function callback:
```suggestion
            const fields: PhrasalTemplateField[] = parentPhrasalTemplateResponseValuePhraseSchema?.value?.fields || { type: '' };

            const isMoreThanOneSentence = fields.filter(({ type }) => type === 'sentence').length > 1;
```